devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -132,7 +132,9 @@ def excludeParameterFromSync(preference){
 def addToAssociationGroupIfNeeded() {
 	def cmds = []
 	if (zwaveInfo?.model?.equals("0052")) {
-		cmds += encap(zwave.associationV2.associationSet(groupingIdentifier: 2, nodeId: [zwaveHubNodeId]))
```

Comment:
Why is this being removed?

Reply:
Everything is works correctly when the device is in group 1